po/sl.po

```diff
@@ -0,0 +1,6238 @@
+msgid "Bundle filename must be specified"
+msgstr "Določiti je treba ime datoteke paketa"
+
+#: app/flatpak-builtins-install.c:150
+#, fuzzy
+msgid "Remote bundles are not supported"
+msgstr "Oddaljeni paketi niso podprti"
+
+#: app/flatpak-builtins-install.c:207
+msgid "Filename or uri must be specified"
+msgstr "Navesti je treba ime datoteke ali uri"
+
+#: app/flatpak-builtins-install.c:287
+#, fuzzy
+msgid "[LOCATION/REMOTE] [REF…] - Install applications or runtimes"
+msgstr ""
+"[LOKACIJA/DALJINEC] [REF...] - Namestite aplikacije ali izvajalne datoteke"
+
+#: app/flatpak-builtins-install.c:313
+#, fuzzy
+msgid "At least one REF must be specified"
+msgstr "Določiti je treba vsaj en REF"
+
+#: app/flatpak-builtins-install.c:324
+msgid "Looking for matches…\n"
+msgstr "Iskanje zadetkov ...\n"
+
+#: app/flatpak-builtins-install.c:439
+#, fuzzy, c-format
+msgid "No remote refs found similar to ‘%s’"
+msgstr "Noben oddaljeni refs ni bil podoben \"%s\""
+
+#: app/flatpak-builtins-install.c:504 app/flatpak-builtins-uninstall.c:293
+#: common/flatpak-ref-utils.c:690 common/flatpak-ref-utils.c:1596
+#, c-format
+msgid "Invalid branch %s: %s"
+msgstr "Neveljavna veja %s: %s"
+
+#: app/flatpak-builtins-install.c:531
+#, fuzzy, c-format
+msgid "Nothing matches %s in local repository for remote %s"
+msgstr "Nič se ne ujema %s v lokalnem skladišču za oddaljene %s"
+
+#: app/flatpak-builtins-install.c:533
+#, fuzzy, c-format
+msgid "Nothing matches %s in remote %s"
+msgstr "Nič se ne ujema %s v oddaljenih %s"
+
+#: app/flatpak-builtins-install.c:554
+#, fuzzy, c-format
+msgid "Skipping: %s\n"
+msgstr "Preskoči: %s\n"
+
+#: app/flatpak-builtins-kill.c:68
+#, fuzzy, c-format
+msgid "%s is not running"
+msgstr "%s se ne izvaja"
+
+#: app/flatpak-builtins-kill.c:82
+#, fuzzy
+msgid "INSTANCE - Stop a running application"
+msgstr "INSTANCE – Zaustavitev tekočih aplikacij"
+
+#: app/flatpak-builtins-kill.c:90 app/flatpak-builtins-ps.c:252
+#, fuzzy
+msgid "Extra arguments given"
+msgstr "Podani dodatni argumenti"
+
+#: app/flatpak-builtins-kill.c:96
+#, fuzzy
+msgid "Must specify the app to kill"
+msgstr "Določiti morate aplikacijo za"
+
+#: app/flatpak-builtins-list.c:46
+#, fuzzy
+msgid "Show extra information"
+msgstr "Pokaži dodatne informacije"
+
+#: app/flatpak-builtins-list.c:47
+#, fuzzy
+msgid "List installed runtimes"
+msgstr "Seznam nameščenih med izvajanjem"
+
+#: app/flatpak-builtins-list.c:48
+#, fuzzy
+msgid "List installed applications"
+msgstr "Seznam nameščenih aplikacij"
+
+#: app/flatpak-builtins-list.c:49
+#, fuzzy
+msgid "Arch to show"
+msgstr "Arch za prikaz"
+
+#: app/flatpak-builtins-list.c:50 app/flatpak-builtins-remote-ls.c:55
+#, fuzzy
+msgid "List all refs (including locale/debug)"
+msgstr "Naštejte vse refs (vključno z lokalnimi ali debug)"
+
+#: app/flatpak-builtins-list.c:51 app/flatpak-builtins-remote-ls.c:56
+#, fuzzy
+msgid "List all applications using RUNTIME"
+msgstr "Na seznam vseh aplikacij z uporabo PROGRAMA RUNTIME"
```

Comment:
`PROGRAMA` shouldn't be capitalized, this refers to an argument named `RUNTIME`.